doc/_utils/redirects.py

```diff
@@ -4,455 +4,474 @@
+    ("protocols/wifi/sniffer_rx_operation","protocols/wifi/advanced_modes/sniffer_rx_operation"), # Raw IEEE 802.11 packet reception using Monitor mode
+    ("device_guides/working_with_nrf/nrf70/developing/regulatory_support","protocols/wifi/regulatory_support"), # Operating with regulatory support
+    ("device_guides/working_with_nrf/nrf70/developing/debugging","protocols/wifi/debugging"), # Debugging
+    ("ug_zigbee","protocols/zigbee/index"), # Zigbee (landing)
+    ("ug_zigbee_qsg","protocols/zigbee/qsg"), # Zigbee quick start guide
+    ("ug_zigbee_supported_features","protocols/zigbee/supported_features"), # Supported Zigbee features
+    ("ug_zigbee_architectures","protocols/zigbee/architectures"), # Zigbee architectures
+    ("ug_zigbee_commissioning","protocols/zigbee/commissioning"), # Zigbee commissioning
+    ("ug_zigbee_memory","protocols/zigbee/memory"), # Zigbee memory requirements
+    ("ug_zigbee_configuring","protocols/zigbee/configuring"), # Configuring Zigbee in nRF Connect SDK
+    ("ug_zigbee_configuring_libraries","protocols/zigbee/configuring_libraries"), # Configuring Zigbee libraries in nRF Connect SDK
+    ("ug_zigbee_configuring_zboss_traces","protocols/zigbee/configuring_zboss_traces"), # Configuring ZBOSS traces in nRF Connect SDK
+    ("ug_zigbee_adding_clusters","protocols/zigbee/adding_clusters"), # Adding ZCL clusters to application (removed after 2.5.0)
+    ("ug_zigbee_other_ecosystems","protocols/zigbee/other_ecosystems"), # Configuring Zigbee samples for other ecosystems
+    ("ug_zigbee_tools","protocols/zigbee/tools"), # Zigbee tools
+    ("applications/nrf5340_audio/README","applications/nrf5340_audio/index"), # nRF5340 Audio applications (landing)
+    ("samples/samples_bl","samples/bl"), # Bluetooth samples (landing)
+    ("samples/samples_nrf9160","samples/cellular"), # Cellular samples (landing)
+    ("samples/samples_crypto","samples/crypto"), # Cryptography samples (landing)
+    ("samples/samples_edge","samples/edge"), # Edge Impulse samples (landing)
+    ("samples/samples_gazell","samples/gazell"), # Gazell samples (landing)
+    ("samples/samples_matter","samples/matter"), # Matter samples (landing)
+    ("samples/samples_nfc","samples/nfc"), # NFC samples (landing)
+    ("samples/samples_nrf5340","samples/nrf5340"), # nRF5340 samples (landing)
+    ("samples/samples_thread","samples/thread"), # Thread samples (landing)
+    ("samples/samples_tfm","samples/tfm"), # Trusted Firmware-M (TF-M) samples (landing)
+    ("samples/samples_wifi","samples/wifi"), # Wi-Fi samples (landing)
+    ("samples/wifi/sr_coex/README","samples/wifi/ble_coex/README"), # Wi-Fi: Bluetooth LE coexistence
+    ("samples/samples_zigbee","samples/zigbee"), # Zigbee samples (landing)
+    ("samples/samples_other","samples/other"), # Other samples (landing)
+    ("libraries/bin/bt_ll_acs_nrf53/index", "nrfxlib/softdevice_controller/doc/isochronous_channels"), # LE Audio controller for nRF5340 (removed for 2.7.0)
```

Comment:
This redirection is not working, it resolves to https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16834/nrf/nrfxlib/softdevice_controller/doc/isochronous_channels.html

Reply:
@gmarull , is it OK to use `../` in these tuples?

Reply:
Seems to be working.